lib/floki/selector/attribute_selector.ex

```diff
@@ -103,7 +103,7 @@ defmodule Floki.Selector.AttributeSelector do
 
   def match?(attributes, s = %AttributeSelector{match_type: :includes, value: value}) do
     get_value(s.attribute, attributes)
-    |> String.split(~r/\s+/)
+    |> String.split([" ", "\t", "\n"])
```

Comment:
And here :)

Reply:
fa64b01261028d47eb5d41c1f6aab840a781f636